packages/dds/sequence/src/sequence.ts

```diff
@@ -12,7 +12,7 @@ import {
 	IFluidDataStoreRuntime,
 } from "@fluidframework/datastore-definitions";
 import {
-	// eslint-disable-next-line import/no-deprecated
+	/* eslint-disable import/no-deprecated */
```

Comment:
Nit: this change means that new deprecated imports can be added from merge-tree without explicit awareness. As clunky as it seems, per-line statements are the way to go for these disables IMO.